dracut-init.sh

```diff
@@ -559,46 +559,6 @@ build_ld_cache() {
     fi
 }
 
-prepare_udev_rules() {
```

Comment:
The `prepare_udev_rules` function is also called from 3rd party modules (see for example https://github.com/leahneukirchen/hrmpf/blob/master/dracut/module-setup.sh#L22). I'd issue a deprecation warning on this function and postpone its removal.

Reply:
I'd write it just after entering the function. Something like:

```patch
 prepare_udev_rules() {
+    dwarn "prepare_udev_rules: YOUR DEPRECATION MESSAGE HERE"
+
     if [ -z "$UDEVVERSION" ]; then
         UDEVVERSION=$(udevadm --version)
         export UDEVVERSION
```